community/neo4j/src/test/java/upgrade/StoreUpgraderInterruptionTestIT.java

```diff
@@ -129,10 +132,12 @@ public void migrate( File sourceStoreDir, File targetStoreDir, SchemaIndexProvid
         assertTrue( checkNeoStoreHasLatestVersion( check, workingDirectory ) );
         assertTrue( allStoreFilesHaveNoTrailer( fs, workingDirectory ) );
 
+        startStopDatabase( workingDirectory );
```

Comment:
Ok, this is for migration from 1.9 to build label scan store. Would be good to have a comment about why this method is needed.